### PR TITLE
disk: fix ensureLVM for partition tables without /boot

### PR DESCRIPTION
**Pull request by gicmo**
When the partition table did not have a boot partition, we created it but then _unconditionally_ returned, which meant that we did not the LVM skeleton and wrap the root partition. Properly handle this case and also re-initialize the `rootPath` in this case since the underlying `Partition[]` array in `PartitionTable` object.
Add an extra blueprint with only one customisation which exposes this bug.

### CODE REVIEW

**Comment by achilleas-k** [comment]
This needs an additional fix for ensuring `/usr` and `/` are above a minimum size.
Fix incoming (with manifests).

**Comment by achilleas-k** [comment]
Should be good to go

**Review by atodorov — COMMENTED** [review]
FTR let's extend filesystem.sh to assert against the partition type of the customized partitions, see proposal & commands in https://bugzilla.redhat.com/show_bug.cgi?id=2079383#c3

**Comment by achilleas-k** [comment]
New rebase: See last 2 commits

**Review by thozza — APPROVED** [review]
Looks great 👍

File: internal/disk/disk_test.go [hunk 1]
```diff
@@ -502,6 +502,121 @@ func TestCreatePartitionTableLVMify(t *testing.T) {
...
+			ExpectedMinSizes: map[string]uint64{
+				"/opt": 10 * GiB,
+				"/":    3 * GiB,
+			},
+		},
+	}
+
+	for idx, tc := range testCases {
+		{ // without LVM
+			mpt, err := NewPartitionTable(&pt, tc.Blueprint, uint64(3*GiB), false, rng)
```
[the remark this answers is not in the document]
Comment (thozza) [reply internal/disk/disk_test.go:591]: ahh, I didn't notice the casting. 🤦  nevermind, this version is definitely good 😉

**Pull request merged**